src/status_im/ui/screens/chat/photos.cljs

```diff
@@ -12,7 +12,8 @@
      [react/view {:style (style/photo-container size)}
       [fast-image/fast-image {:source              {:uri photo-path}
                               :style               (style/photo size)
-                              :accessibility-label (or accessibility-label :chat-icon)}]
+                              :accessibility-label (or accessibility-label :chat-icon)
+                              :fallback true}]
```

Comment:
Align map

Reply:
thought `make lint-fix` would fix this ):